documentation/src/docs/controller-service.md

```diff
@@ -544,6 +529,57 @@ Streams to go on concurrently. Workflows like _scale, truncation, seal,
 update_ and _delete Stream_ are implemented for processing on the request
 Event Processor.
 
+## Periodic Background job framework 
+
+To enable some scenarios, we may need the background workers to
+periodically work on each of the Streams in our cluster to perform
```

Comment:
Now you write Pravega abstractions/components like `Stream` or `Controller` with capitals. I like it and is how most of the documentation is actually doing. The problem is that most parts of this document do not keep this style.